lib/utils/classes.py

```diff
@@ -1,5 +1,9 @@
 import datetime
 import argparse
+import requests
+import pandas as pd
```

Comment:
I see you are using some external libraries. Can you add all external libraries to the root directory in a file called `requirements.txt` - one per line?

That way when someone wants to set the project up, they can just run `pip3 install -r requirements.txt`

😄

Reply:
Done